wcivf/tests/test_urls.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+from dc_utils.tests.helpers import validate_html
+from django.urls import reverse
+from people.tests.factories import PersonFactory, PersonPostFactory
+from parties.tests.factories import PartyFactory
+from elections.tests.factories import (
+    ElectionFactory,
+    PostFactory,
+    PostElectionFactory,
+)
+
+
+class TestHtml:
+    @pytest.fixture
+    def urls(self):
+        return [
+            reverse("home_view"),
+            reverse("about_view"),
+            reverse("standing_as_a_candidate"),
+            reverse("opensearch"),
+            reverse("status_check_view"),
+            reverse("privacy_view"),
+            PersonPostFactory(
+                election=ElectionFactory()
+            ).person.get_absolute_url(),
+            reverse(
+                "email_person_view",
+                kwargs={"pk": PersonFactory().pk, "ignored_slug": "sadiq-khan"},
+            ),
+            reverse("parties_view"),
+            PartyFactory().get_absolute_url(),
+            reverse("elections_view"),
+            ElectionFactory().get_absolute_url(),
+            reverse(
+                "redirect_election_view",
```

Comment:
Not sure if you have seen but the tests are failing attempting to build this URL https://app.circleci.com/pipelines/github/DemocracyClub/WhoCanIVoteFor/1026/workflows/7da6c9e9-407d-45a9-aa08-963d6e132b70/jobs/1849. Do you have `pre-commit` setup (check the readme) as if you did I would have expected the test to fail locally before pushing?

Although I dont think you need to test this url as its purpose seems to be to return a redirect to the ElectionView, which is already tested on line 33. One that is missing however is a test for `PostElectionView` - you could test this by using the `PostElectionFactory().get_absolute_url()`